common/src/main/scala/com/aliyun/emr/rss/common/protocol/message/ControlMessages.scala

```diff
@@ -80,15 +80,21 @@ object ControlMessages extends Logging {
       fetchPort: Int,
       replicatePort: Int,
       disks: Map[String, DiskInfo],
+      userResourceUsage: Map[UserIdentifier, ResourceConsumption],
```

Comment:
As I have mentioned before, is it necessary?

Reply:
We plan to achieve recover user usage from levelDB in #642 , so if there is recovered user data, I think we may need to bring this information when registering worker.

Reply:
OK.